apps/server/src/modules/deletion/controller/deletion-requests.controller.ts

```diff
@@ -0,0 +1,53 @@
+import { Body, Controller, Delete, Get, HttpCode, Param, Post, UseGuards } from '@nestjs/common';
+import { ApiOperation, ApiResponse, ApiTags } from '@nestjs/swagger';
+import { AuthGuard } from '@nestjs/passport';
+import { DeletionRequestUc } from '../uc';
+import { DeletionRequestLogResponse, DeletionRequestBodyProps, DeletionRequestResponse } from './dto';
+
+@ApiTags('DeletionRequests')
+@UseGuards(AuthGuard('api-key'))
+@Controller('deletionRequests')
+export class DeletionRequestsController {
+	constructor(private readonly deletionRequestUc: DeletionRequestUc) {}
+
+	@Post()
+	@HttpCode(202)
+	@ApiOperation({
+		summary: '"Queueing" a deletion request',
+	})
+	@ApiResponse({
+		status: 202,
+		type: DeletionRequestResponse,
+		description: 'Returns identifier of the deletion request and when deletion is planned at',
+	})
+	async createDeletionRequests(
+		@Body() deletionRequestBody: DeletionRequestBodyProps
+	): Promise<DeletionRequestResponse> {
+		return this.deletionRequestUc.createDeletionRequest(deletionRequestBody);
+	}
+
+	@Get(':requestId')
+	@ApiOperation({
```

Comment:
I think you've missed the `@HttpCode(200)` annotation here?

Reply:
by default it returns 200 but I can add it